siliconcompiler/scheduler/send_messages.py

```
@@ -219,24 +219,24 @@ def send(chip, msg_type, step, index):
             smtp_server.login(cred["username"], cred["password"])
             do_send = True
         except smtplib.SMTPAuthenticationError as e:
-            chip.logger.error(f'Unable to authenticate to email server: {e}')
+            project.logger.error(f'Unable to authenticate to email server: {e}')
```

Ruff (0.13.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)